### ISSUE TITLE
When opening a console on a VMware Fusion vm I got this error

### ISSUE THREAD

**Issue by noplay** [GNS3/gns3-gui#664]
```
 stty raw; nc -U "/var/folders/3s/r2wbv07n7wg4vrsn874lmxxh0000gn/T/gns3_vmware/c19f9587-d8d5-420a-8f2f-a24c82338eb5"; exit
nc: invalid option -- U
                       Try `nc --help' for more information.
                                                            logout

[Opération terminée]
```

**Comment by noplay** [comment]
Ok the option is in the man page but not in the binary fun :D

**Comment by noplay** [comment]
Thanks to apple :( I have dropped nc but people need to manually install socat. I think we can accept that because nobody complain about that  feature not working.

**Comment by ehlers** [comment]
I have used nc -U on Mavericks (10.9), now on El Capitan (10.11), I have skipped Yosemite (10.10). No problems with at all, works without problems.

Sure you don't have a non-apple nc outside /usr/bin ,e.g. something in /usr/local/bin ?

**Comment by noplay** [comment]
```
➤  whereis nc
/usr/bin/nc
➤  nc -U
nc: invalid option -- U
Try `nc --help' for more information.
```

But /usr/bin/nc work ...

**Comment by noplay** [comment]
Il will restore nc and force the full path

**Comment by ehlers** [comment]
Do you have an alias or shell function named nc ? Otherwise it's rather strange, that /usr/bin/nc works, but nc doesn't. On my systems nc -U works fine, even without full path.

**Comment by noplay** [comment]
I checked and I have a GNU nc from homebrew.  Perhaps whereis lie and the nc in /usr/local/bin is used.

Only BSD nc support socket... Hard coding the path will avoid issues for user like me.

**Comment by ehlers** [comment]
I had a deeper look in the whereis utility, that's what the man page says:

The whereis utility checks the **standard** binary directories for the specified programs, printing out the paths of any it finds.

The path searched is the string returned by the sysctl(8) utility for the ``user.cs_path'' string.

```
/Users/behlers# sysctl user.cs_path
user.cs_path: /usr/bin:/bin:/usr/sbin:/sbin
```

So I learned that whereis is useless to find executables in $PATH, better to use "which -a" for that. On my OS X system I have replaced whereis by an alias that points to "which -a", as I don't like the current implementation of whereis.
